docs/DeveloperGuide.md

```diff
@@ -394,6 +403,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   - 1a1. System informs Actor that they have no contacts that match.
 
     Use case ends.
+    
+**Use case: UC6 - Add notes to a person contact**
+
+**MSS**
+
+1.  Actor performs <u>list all people (UC2)</u>.
```

Comment:
Can double check do we actually need this line since our application the user will never need to list all people?